samples/ballerina-by-example/examples/if-else/if-else.sh

```diff
@@ -0,0 +1,9 @@
+# To run the program, create a file named `if-else.bal`
+# and run the command below.
+
+$ $BALLERINA_HOME/bin/ballerina if-else.bal
+a is greater
+This is number 10
+This is zero
+
+# Next example:
```

Comment:
@dilinisg  Remove this line. Next sample linking happens through the examples.txt file entry.